openweathermap-ios/openweathermap-ios/Weather/Model/WeatherData.swift

```diff
@@ -0,0 +1,14 @@
+
+struct WeatherData: Codable {
+    let name: String
```

Comment:
Who is this name for?

Reply:
In api name is the name of the city. There is no way to change or access the name of the city.

![image](https://user-images.githubusercontent.com/21261476/79124010-ccdc7d80-7d71-11ea-871d-956d03ed37fb.png)

Reply:
Ok, you are respecting the Response properties. It's fine.